_includes/pagination.html

```diff
@@ -4,7 +4,7 @@
   {% else %}
     {% assign currentOffset = 2 %}
   {% endif %}
-  <nav class="u-containProse" role="menu" aria-label="Pagination">
+  <nav class="u-containProse" role="navigation">
```

Comment:
> Question: Was aria-label removed because the element is understandable without? 

Great question @tylersticka! I was just looking at that again and realized that should not have been removed. Good catch! :)

> To make it easier to find menus, they should be labeled according to their individual function. This helps to distinguish between multiple navigational menus on a page. Such labels can be invisible to users that experience the website visually. Such labels can also be used by assistive technology to enable jumping directly to navigation landmarks or headings.

More: https://www.w3.org/WAI/tutorials/menus/structure/#labeling-navigations